src/main/java/mike/cli/CommandLine.java

```diff
@@ -154,36 +161,50 @@ public static void commandInterface() {
 						} else {
 							System.out.println("\nCreate relationship failed. Make sure the classes exist and that it is not a duplicate.\n");							
 						}
+					} else if (commands[1].equals("param")) {
+						if (commands.length != 6) {
+							System.out.println(errorMessage + commandUsage[6] + "\n");
+						} 
```

Comment:
Need to put a break at the end of this if statement. It goes onto the next if, trying to call create parameter with the wrong number of arguments and crashes.

Reply:
Resolved in most recent push 👍